### DISCUSSION TITLE
Another terminal text editor based on Scintilla

### DISCUSSION THREAD

**Discussion by magiblot** [orbitalquark/scinterm#4] [General]
Hi Mitchell,

I'm writing to you because I see you still maintain Scinterm. Last summer I tried to use Scintilla in a terminal application as an experiment, and this was the result: https://github.com/magiblot/turbo.

Scinterm was an important source of inspiration for me to understand how Scintilla worked and to figure out how to get it to do what I needed. So I wanted to thank you personally.

The Turbo text editor relies on a separate framework for the user interface, so it does not need to hardcode the location of UI elements nor deal with terminal capabilities. In other words, the class that specializes `ScintillaBase` does not have the responsibility of controlling the UI. Hopefully it may serve you as a source of inspiration as well.

Cheers.

**Comment by orbitalquark** [comment]
Hi Magiblot, thanks for reaching out and for the kind words. I saw Turbo a month ago and noticed you had effectively written a new platform for Scintilla. Nice work!
